### PR TITLE
Add exec utils for goock

### PR DESCRIPTION
**Pull request by peter-wangxu**
These files are imported from kubernetes.
Also draft the initial README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,39 @@
...
+Goock aims at providing a easy-to-use library for block device discovering and
+management. This project is inspired by OpenStack project
+[os-brick](https://github.com/openstack/os-brick).
+
+### Features
+
+* Discovery iSCSI transport protocol.
+* Discovery FibreChanel transport protocol.
+* Removal of devices from a host
+* Multipath support
```
Comment (LianWaiYuChanChan) [inline README.md]: This is very cool feature.

File: goock/exec.go [hunk 2]
```diff
@@ -0,0 +1,169 @@
+/*
+Copyright 2014 The Kubernetes Authors.
```
Comment (LianWaiYuChanChan) [inline goock/exec.go]: Should we change the Copyright? Just a little comments. You can ignore it. :)

Comment (LianWaiYuChanChan) [inline goock/exec.go]: Just a question: shall we remove all words about Kubernates?

Comment (peter-wangxu) [inline goock/exec.go]: Will remove all words about kubernetes.

File: README.md (hunk 1 above)
Comment (peter-wangxu) [inline README.md]: a long way to go...

**Pull request merged**